README.md

```diff
@@ -1,3 +1,11 @@
-## Immutable Ledger used in Plenum.. 
+## Immutable Ledger used in Plenum. 
+
+This codebase provides a very simple, python-based immutable distributed
```

Comment:
This ledger is not distributed. It is an immutable ordered log of transactions (write operations) backed by merkle tree providing efficient mechanism to generate verifiable proofs of presence and consistency of data.